sdk/tests/deqp/functional/gles3/es3fShaderCommonFunctionTests.js

```diff
@@ -1144,9 +1150,13 @@ goog.scope(function() {
         for (var compNdx = 0; compNdx < scalarSize; compNdx++) {
             in0 = inputs[0][compNdx];
             out0 = outputs[0][compNdx];
-            refOut0 = tcuFloat.newFloat32(in0).bits();
-            ulpDiff = Math.abs(Math.floor(in0) - Math.floor(out0));
 
+            // Convert int to float
+            var view = new DataView(new ArrayBuffer(4));
+            view.setUint32(0, in0, true);
```

Comment:
out0 is alread float type. No data type change for "const float		out0		= ((const float*)outputs[0])[compNdx];" While for in0 it is interpreted as float from int.